### DISCUSSION TITLE
TIL that feature requests are moved to discussions

### DISCUSSION THREAD

**Discussion by ConsoleTVs** [vercel/next.js#16492] [Help] [answered]
**Notice:** This has been solved, as the original post was about a possible deletion of an issue. Turns out the issue was moved to discussions without notice.

~~I made a very long feature request a few days ago, full of examples, descriptions and quality of life improvements for developers. It got a few thumbs up in the very first hours and no comments. And apparently it got ~~closed~~... I mean deleted! There's no trace of it. Not a single line of feedback or comment.~~

~~I spent around 2h writing the examples and text and gathering information for it. I completely think that an issue marked as a "Feature Request" should not be treated as such, specially after people marking it as "thumbs up"?~~

~~Am I missing something here or what? Can't you guys simply say "Hey we're not interested in this nor we'll be building it"? What's the point in deleting an entire handcrafted issue from the repo otherwise? Aren't you interested in people's feedback?~~

**Comment by balazsorban44** [comment]
I think I have seen your feature request not so long ago, and even subscribed to it to hear about when something happens.

It indeed sounds strange that it disappeared, but I am certain there is a good explanation. I have had direct help from @Timer before, so I am pretty sure they listen well to their community. I am interested in the response also, so I will follow this discussion.

**Reply by ConsoleTVs** [reply]
Thank you @balazsorban44 . I hope there's indeed an explanation for it. I wasn't expecting an issue to just "disappear" out of nowhere without at least a comment from a maintainer.

I just made some suggestions about a possible persistent layout system, a nested _app, verifications and middlewares. 

Let's see if some maintainer is able to get some insight in here.

**Comment by mattcarlotta** [comment, marked as answer]
@ConsoleTVs Are you referring to [this post](https://github.com/vercel/next.js/discussions/16400)?

**Reply by ConsoleTVs** [reply]
Yes. Why has this been moved from Issues to Discussions without prior notice?

**Reply by ConsoleTVs** [reply]
Thank you for finding it, I could not even find it through the reference at https://github.com/vercel/next.js/issues/8193. I still have no idea why this was moving without notice since github does not warn about such changes.

**Reply by mattcarlotta** [reply]
About 2-3 months ago, it appears RFCs were moved to discussions to separate them from issues.

**Reply by ConsoleTVs** [reply]
Thank you for clarification! Maybe they should remove this then...

![image](https://user-images.githubusercontent.com/6124435/90981730-01074c80-e563-11ea-8a25-520f2605ece6.png)

**Reply by timneutkens** [reply]
Discussions do not have templates yet so we're handling them by manually converting. The issue url is redirected automatically by GitHub. Will relay to the github team that it's not clear yet that an issue gets moved

**Reply by ConsoleTVs** [reply]
Thank you for your time Tim. Perhaps, a middle ground would be to add that clarification in the issue pre-filled text for the Feature Request template: 

![image](https://user-images.githubusercontent.com/6124435/90983469-cc998d80-e56e-11ea-92b6-a2ed4f91e476.png)
